docs/source/tutorial_r.rst

```diff
@@ -0,0 +1,106 @@
+Pywps with R
+============
+The following shows how you can wrap R software with pyWPS.
+
+.. _examples:
+
+Examples of R Birds
+...........................
+* `pcic/quail <https://github.com/pacificclimate/quail>`_
+
+  * `wps_climdex_gsl.py <https://github.com/pacificclimate/quail/blob/master/quail/processes/wps_climdex_gsl.py>`_
+* `pcic/chickadee <https://github.com/pacificclimate/chickadee>`_
+
+  * `wps_BCCAQ.py <https://github.com/pacificclimate/chickadee/blob/master/chickadee/processes/wps_BCCAQ.py>`_ 
+
+.. _rpy2:
+
+Rpy2
+....
+There’s several R-to-python Python libraries but `Rpy2 <https://rpy2.github.io/doc/latest/html/index.html>`_ is probably the most well documented and most frequently used. As long as it is installed, a R library can be accessed with :code:`importr([package-name])`. Since R :code:`base` and :code:`utils` are installed with :code:`rpy2` you can import them:
+
+.. code-block:: Python
+   
+   from rpy2.robjects.packages import importr
+   base = importr("base")
+   utils = importr("utils")
+   
+Then you can use functions from that package with :code:`package.function_name()`. If the R function name has a period :code:`.` it is replaced with an underscore :code:`_` in python.
+ 
+.. code-block:: Python
+
+   base.all(True)
+   base.all_equal("hello","world") # all.equal() in R
+ 
+You can execute any regular R code as a string passed to :code:`robjects.r()`
+
+.. code-block:: Python
+
+   from rpy2 import robjects
+   count = robjects.r("c(1,2,3)")
+   robjects.r("all(T)")
+   
+You can also access R functions with the syntax :code:`robjects.r["function.name"]` if you want to avoid the import step.
+
+.. code-block:: Python
+
+   robjects.r["save"](count, file=output_path)
+   
+Install another package with Rpy2 and use the functions form that package...
+
+.. code-block:: Python
+
+   utils.install_packages("climdex.pcic")
+   climdex_pcic = importr("climdex.pcic")
+   climdex.climdex_gsl(climdexInput, gsl_mode)
+   
+.. _io:
+
+I/O
```

Comment:
Any particular issue with LiteralInputs ? E.g. datetime objects.

Reply:
From my experience, Rpy2 handles the conversion, so they can just be passed regularly without additional handling.